pkg/crc/machine/start.go

```diff
@@ -524,6 +524,9 @@ func createHost(api libmachine.API, machineConfig config.MachineConfig) error {
 		return fmt.Errorf("Error in driver during machine creation: %s", err)
 	}
 
+	if err := cluster.GenerateKubeAdminUserPassword(); err != nil {
```

Comment:
My understanding is that this kubeadmin password change will be fixing an error which was seen by some users? Can this be mentioned in the commit log?

Reply:
yes added.
```
Cannot create cluster configuration: Error reading kubeadmin password from bundle
```